File: academy/trainer-service/src/main/java/com/easyskillup/trainerservice/model/Trainer.java
package com.easyskillup.trainerservice.model;

import java.util.HashSet;
import java.util.Set;

import javax.annotation.Generated;
import javax.persistence.ElementCollection;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.OneToMany;
import javax.persistence.Table;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Entity
@Table(name = "trainers")

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class Trainer {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer trainerId;

    private String trainerName;

    private String trainerEmail;

    private String track;

    @ElementCollection
    private Set<String> specializations = new HashSet<>();

}

File: anuglar/training-app/src/app/models/trainer.ts
export class Trainer {

    trainerName: string = '';
    trainerEmail: string = '';
    track: string = '';

    constructor(trainerName: string, trainerEmail: string, track: string) {

        this.track = track;
        this.trainerEmail = trainerEmail;
        this.trainerName = trainerName;

    }

}
